erpnext/patches/v13_0/rename_lost_reason_detail.py

```diff
@@ -0,0 +1,13 @@
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+    if frappe.db.exists("DocType", "Opportunity Lost Reason Detail") and frappe.db.exists("DocType", "Quotation Lost Reason Detail") and frappe.db.exists("DocType", "Lost Reason Detail"):
```

Comment:
why checking db.exists for "Opportunity Lost Reason Detail" and "Quotation Lost Reason Detail"? Those are new table, should be reloaded with `frappe.reload_doc`